### PR TITLE
BUG: fix annualization of alpha

### PR DESCRIPTION
**Pull request by vikram-narayan**
For #59, annualize alpha correctly

### CODE REVIEW

**Comment by twiecki** [comment]
This looks right. But would hope that some tests catch this we need to change.

**Comment by vikram-narayan** [comment]
@twiecki just updated `test_stats.py` to fix the failures - mind taking a quick look at those as well? Mostly just updating numbers

File: empyrical/tests/test_stats.py [hunk 1]
```diff
@@ -1111,7 +1114,7 @@ def test_roll_up_capture(self, returns, factor_returns, window, expected):
         (empty_returns, simple_benchmark, (np.nan, np.nan)),
         (one_return, one_return, (np.nan, np.nan)),
         (mixed_returns[1:], negative_returns[1:],
-         (-8.306666666666668, -0.71296296296296313)),
+         (-0.9997853834885004, -0.71296296296296313)),
```
Comment (twiecki) [inline empyrical/tests/test_stats.py]: This already shows why the previous result was wrong.

**Comment by twiecki** [comment]
LGTM. We should probably alert others to this bugfix.

File: empyrical/stats.py [hunk 2]
```diff
@@ -818,7 +818,7 @@ def alpha_aligned(returns, factor_returns, risk_free=0.0, period=DAILY,
     adj_factor_returns = _adjust_returns(factor_returns, risk_free)
     alpha_series = adj_returns - (_beta * adj_factor_returns)
 
-    return nanmean(alpha_series) * ann_factor
+    return (nanmean(alpha_series) + 1) ** ann_factor - 1
```
Comment (yankees714) [inline empyrical/stats.py]: What would you think about breaking this formula out into a helper function? Feels like that would make it clearer what's happening here, and less error prone.

(At the least a comment to the effect of #59 would be good)

Comment (vikram-narayan) [reply empyrical/stats.py]: added an `annualize` helper in utils, and used that here

File: empyrical/utils.py [hunk 3]
```diff
@@ -443,3 +443,10 @@ def default_returns_func(symbol, start=None, end=None):
         rets = get_symbol_returns_from_yahoo(symbol, start=start, end=end)
 
     return rets[symbol]
+
+
+def annualize(stat, ann_factor=252):
```
[the remark this answers is not in the document]
Comment (chrismyles4) [reply empyrical/utils.py]: Nice catch, I agree

[the remark this answers is not in the document]
Comment (vikram-narayan) [reply empyrical/utils.py]: 👍  renamed to `annualize_compounding`

**Comment by chrismyles4** [comment]
FYI I have walked Vikram's analysis of where this code is used to the PM team, Dan and Rich. We are good to go, they will be adding a forum post to explain the change.

> I agree that a forum announcement is appropriate and will be a non issue.  "we love open source software, our calculations are all out there, if you see any mistakes we're all better off when we fix them."

**Comment by yankees714** [comment]
LGTM

**Comment by twiecki** [comment]
Can we merge this?

**Comment by vikram-narayan** [comment]
We chose to wait for when we have the bandwidth to write a blog post announcing the change. Will follow up on that today

File: empyrical/stats.py [hunk 4]
```diff
@@ -1279,6 +1280,9 @@ def alpha_aligned(returns,
     adj_factor_returns = _adjust_returns(factor_returns, risk_free)
     alpha_series = adj_returns - (_beta * adj_factor_returns)
 
+<<<<<<< HEAD
```
Comment (twiecki) [inline empyrical/stats.py]: conflict

**Comment by twiecki** [comment]
I think there is still a bug here, as the failing tests show. Seems like it always returns 0 or nan?

**Comment by twiecki** [comment]
Thanks @gmanoim-quantopian !

**Pull request merged**